Cython/Compiler/Nodes.py

```diff
@@ -3393,7 +3405,7 @@ def generate_function_definitions(self, env, code):
                 self.return_type.declaration_code(Naming.retval_cname),
                 retval_init))
         code.put_declare_refcount_context()
-        code.put_setup_refcount_context('%s (wrapper)' % self.name)
+        code.put_setup_refcount_context(EncodedString('%s (wrapper)' % self.name))
```

Comment:
Could be that something like this is needed in a couple of more places – might be one of the reasons why the `AttributeError`s appeared for you.